tools/ci/make_hosts_file.py

```diff
@@ -3,14 +3,16 @@
 
 from ..localpaths import repo_root
 
-from ..serve.serve import load_config, make_hosts_file
+from ..serve.serve import build_config, make_hosts_file
 
 def create_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("address", default="127.0.0.1", nargs="?", help="Address that hosts should point at")
     return parser
 
 def run(**kwargs):
-    config = load_config(os.path.join(repo_root, "config.json"))
+    config_builder = build_config(os.path.join(repo_root, "config.json"),
+                                  ssl={"type":"none"})
 
-    print(make_hosts_file(config, kwargs["address"]))
+    with config_builder.get() as config:
```

Comment:
I might have missed something but where is `ConfigBuilder.get` defined?

Reply:
That's just a bug; the initial iteration had a `.get` method. I'm surprised the tests didn't catch this.

Reply:
Sorry please disregard. I wrote too much code in blinkpy (which uses `frozendict` a lot) to the point I somehow thought PEP 416 was implemented...